sdk/iOS/src/MSCoreDataStore.m

```diff
@@ -65,6 +65,9 @@ -(id) getRecordForTable:(NSString *)table itemId:(NSString *)itemId asDictionary
     
     if (item && asDictionary) {
         
+        //Someone, please, test this. Solves the warning, but I don't know it it will work as intended.
+//        NSDictionary *result = [item dictionaryWithValuesForKeys:[[[item entity] attributesByName] allKeys]];
+        
```

Comment:
There is a unit test that the expected columns get out, our unit test is missing a check on unexpected columns though (which is why #779 exists)  If the MSCoreDataStoreTests class is passing post this change, its ok.  We will be revamping this logic shortly anyways as outlined in my proposal for 779 anyways.